### DISCUSSION TITLE
Github reset my password, but I don't have 2FA

### DISCUSSION THREAD

**Discussion by LYingSiMon2** [community/community#131379] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Security and Privacy

### Body

I received an email today：
--------------------
Hello @LYingSiMon,

We recently noticed that your GitHub account had a suspicious login. Out of an abundance of caution, we made the decision to force a password reset for the account associated with this email address.

We have no reason to believe that this was the result of a GitHub vulnerability. While we can’t speak to your particular case, in other similar account takeovers our users have told us that infostealer malware was discovered on their system.

Follow the link below to request a password reset token.

https://github.com/password_reset

To protect your account from unauthorized access, please choose a strong and unique password for your account. We have a help article with some recommendations here:

https://docs.github.com/authentication/keeping-your-account-and-data-secure/creating-a-strong-password

If you have not already done so, we strongly recommend taking the additional step to secure your account with two-factor authentication. The following guide provides step-by-step instructions:

https://docs.github.com/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication

After you sign in, please review your security history for suspicious or malicious activity.

https://github.com/settings/security

Please also check for any suspicious starring activity and any malicious activity on repositories you own or have access to, paying particular attention to repositories with names you do not recognize and any repositories that have recently been created. For more information, read Deleting a repository in the GitHub Docs;

https://docs.github.com/repositories/creating-and-managing-repositories/deleting-a-repository

If you have specific concerns you may contact our support team by submitting a message through our contact form.

https://support.github.com/contact/account

Thanks,

GitHub Support
---------------
I don't have 2FA, so I can't change my password,I have submitted the application：

Ticket ID: 2874667

But I don't know how long it will take. I rely heavily on GitHub, which will cause me great losses. Why can't I use email verification directly?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @LYingSiMon2, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

Thank you!
